openai/openai_object.py

```diff
@@ -247,7 +248,7 @@ async def arequest(
                     self.organization,
                     plain_old_data=plain_old_data,
                 )
-                for line in response
+                async for line in response
```

Comment:
I'm almost sure It's a bug since the response returns `OpenAIResponse` if `stream` is False or an async generator of `OpenAIResponse` if `stream` is True.